tasks/install-FreeBSD.yml

```diff
@@ -4,6 +4,14 @@
   pkgng:
     name: rubygem-fluentd
     state: present
+  register: register_fluentd_rubygem_fluentd
+
+- name: Patch rc.d script
+  patch:
```

Comment:
lineinfile is safer as the rc.d script might be modified in the other version of the package and the patch fails.